docs/api/python/rnn.md

```diff
@@ -0,0 +1,339 @@
+# RNN Cell API
+
+```eval_rst
+.. currentmodule:: mxnet.rnn
+```
+
+## Overview
+
+The `rnn` module includes the recurrent neural network (RNN) cell APIs, a suite of tools for building the symbolic graph of RNN.
+
+## The `rnn` module
+
+### Cell interfaces
+
+```eval_rst
+.. autosummary::
+    :nosignatures:
+
+    BaseRNNCell.__call__
+    BaseRNNCell.unroll
+    BaseRNNCell.reset
+    BaseRNNCell.begin_state
+    BaseRNNCell.unpack_weights
+    BaseRNNCell.pack_weights
+```
+
+The cell API operates on symbols and returns output symbols based on the type of RNN. Take
+Long-short term memory (LSTM) as an example:
+
+```python
+import mxnet as mx
+# Shape of 'step_data' is (batch_size,).
+step_input = mx.symbol.Variable('step_data')
+
+# First we embed our raw input data to be used as LSTM's input.
+embedded_step = mx.symbol.Embedding(data=step_input, \
+                                    input_dim=input_dim, \
+                                    output_dim=embed_dim)
+
+# Then we create an LSTM cell.
+lstm_cell = mx.rnn.LSTMCell(num_hidden=50)
+# Initialize its hidden and memory states.
+# 'begin_state' method takes an initialization function, and uses 'zeros' by default.
+begin_state = lstm_cell.begin_state()
+```
+
+The LSTM cell and other non-fused RNN cells are callable. Calling the cell transforms the input once based on LSTM definition. See this [blog post](http://colah.github.io/posts/2015-08-Understanding-LSTMs/) for a great introduction to LSTM and other RNN.
```

Comment:
Change to: "Calling the cell updates it's state once. This transformation depends on both the current input and the previous hidden state." 

(dropping the based on LSTM definition)